### ISSUE TITLE
Add compute_kernel_config argument to batch_norm

### ISSUE THREAD

**Issue by rmillerTT** [tenstorrent/tt-metal#24167]
### Is your feature request related to a problem? Please describe

Batch norm compute kernel is not configurable like other ops

### Describe the solution you'd like

Add and expose the compute_kernel_config parameter to batch_norm as is done for other ops

**Comment by rmillerTT** [comment]
Related to #23534
